packages/vue3/src/components/Spinner/Spinner.vue

```diff
@@ -0,0 +1,55 @@
+<script setup lang="ts">
+import type { SpinnerType } from './types';
+
+defineProps<{ spinner: SpinnerType }>();
+</script>
+
+<template>
+  <div class="container">
+    <div class="spinner" />
+    <p class="label">
+      {{ spinner.label }}
+    </p>
+  </div>
+</template>
+
+<style lang="scss" scoped>
+.container {
+  display: flex;
+  flex-direction: column;
+}
+.label {
+  font: v-bind(
+    "`var(--text-${spinner.labelFontSize}-regular)` || 'var(--text-md-regular)'"
+  );
+  color: v-bind("spinner.labelColor || 'var(--brand-600)'");
+}
+.spinner {
+  width: v-bind("spinner.size || '50px'");
+  aspect-ratio: 1;
+  border-radius: 50%;
+  border: 8px solid #0000;
```

Comment:
We have a variable for black

Reply:
it's transparent not black should i keep it this way or use 'transparent' ?